test/ros_parameter_handler_test.cpp

```diff
@@ -157,20 +162,22 @@ TEST_F(ROSParameterHandlerTest, test_all_params)
   std::string publish_topic = "zyxwvu";
   float angle_start = 10.5;
   float angle_end = 204.7;
-
+  double x_axis_rotatione = (angle_end - angle_start)/2. + angle_start;
```

Comment:
- [ ] What is this calculation supposed to symbolize?
- [ ] Also is the "e" in `x_axis_rotatione` a typo?

Reply:
x_axis_rotation is set to be in the middle between angle_start and angle_end